notebooks/llm-chatbot/rag-chatbot.ipynb

```diff
@@ -18,25 +18,31 @@
     "- Download and convert the model from a public source using the [OpenVINO integration with Hugging Face Optimum](https://huggingface.co/blog/openvino).\n",
```

Comment:
<p>For me format [model name] [model_card_link] looks unreadable</p><ul><li><strong>bge-small-en-v1.5</strong> <a href="https://huggingface.co/BAAI/bge-small-en-v1.5" target="_blank">model card</a></li></ul><p> </p><p>maybe better to use model name as link of model card?</p><ul><li><a href="https://huggingface.co/BAAI/bge-small-en-v1.5" target="_blank">bge-small-en-v1.5</a></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1848/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
done